index.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const helmet = require('helmet');
+
+const server = express();
+
+const knex = require('knex');
+const knexConfig = require('./knexfile');
+const db = knex(knexConfig.development);
+
+const dishesRoutes = require('./routes/dishesRoutes');
+
+server.use('/dishes', dishesRoutes);
+
+server.use(helmet());
+server.use(express.json());
+
+server.get('/', (req, res) => {
+  db('dishes')
+    .then(toCook => {
+      res.status(200).json(toCook);
+    })
+    .catch(error => res.status(400).json(error));
+});
+
+// server.get('/', (req, res) => {
+//   db('instructions')
+//     .select('steps', 'recipe_id')
+//     .from('intructions').join('recipes').on('id' = 'recipe_id').then(answer => {
+//         res.status(200).json(answer)
+//     })
+//     .catch(error => res.status(400).json(error));
+// });
+
+server.get('/dishes/:id/ingredients', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const dishes = await db('dishes').where({ id: id[0] });
+    const recipes = await db('recipes').where({ recipe_id: id });
+    const eachRecipe = recipes.map(word => `Recipe: ${word.recipe}`);
```

Comment:
Your js is getting good due to the fact you were able to solve this problem of how to return what you want on the fly. You had an object and wanted something out of it, you didn't hesitate at all coming up w/ these maps. Think back to like week 1 where we all were at. Great job.